stdlib/2and3/filecmp.pyi

```diff
@@ -2,16 +2,29 @@
 import sys
 from typing import AnyStr, Callable, Dict, Generic, Iterable, List, Optional, Sequence, Tuple, Union, Text
 
+if sys.version_info >= (3, 6):
+    from pathlib import Path
+
 DEFAULT_IGNORES: List[str]
 
-def cmp(f1: Union[bytes, Text], f2: Union[bytes, Text], shallow: Union[int, bool] = ...) -> bool: ...
-def cmpfiles(a: AnyStr, b: AnyStr, common: Iterable[AnyStr],
-             shallow: Union[int, bool] = ...) -> Tuple[List[AnyStr], List[AnyStr], List[AnyStr]]: ...
+if sys.version_info >= (3, 6):
+    def cmp(f1: Union[bytes, Text, Path], f2: Union[bytes, Text, Path], shallow: Union[int, bool] = ...) -> bool: ...
+    def cmpfiles(a: Union[AnyStr, Path], b: Union[AnyStr, Path], common: Iterable[AnyStr],
```

Comment:
This should be `PathLike[AnyStr]` I think, so that the str/bytes match is right even if you use PathLikes.

Though I think that may run into an annoying mypy bug.

Reply:
I am not sure `pathlib.Path` accepts bytes. I get this error: `TypeError: argument should be a str object or an os.PathLike object returning str, not <class 'bytes'>`

I have added the changes and surprisingly it didn't break anything